### PR TITLE
Optimus exon counts

### PR DESCRIPTION
**Pull request by khajoue2**
* Changed how exon counts are computed in sn_rna mode of Optimus

### CODE REVIEW

**Review by nikellepetrillo — APPROVED** [review]
lgtm!

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 5.3.1
+2022-04-06 (Date of Last Commit)
+
+* Updated STARsoloFastq task in StarAlign.wdl
```
Comment (ekiernan) [inline pipelines/skylab/optimus/Optimus.changelog.md:4]:
```suggestion
* Updated the STARsoloFastq task in StarAlign.wdl to run STARsolo independently with "Gene" COUNTING_MODE when the Optimus input parameter `count_exons` is set to true.
```

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.2.4
+2022-04-06 (Date of Last Commit)
+
+* Updated STARsoloFastq task in StarAlign.wdl
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:4]:
```suggestion
* Updated STARsoloFastq task in StarAlign.wdl; this change does not affect the Mutl-snSS2 workflow.
```

**Review by ekiernan — APPROVED** [review]
LGTM - added two suggestions for the changelogs. Documentation changes related to this PR are on a separate branch.

**Pull request merged**